app/templates/services/dashboard.html

```diff
@@ -2,58 +2,86 @@
 {% from 'macros/_breadcrumbs.html' import breadcrumb %}
 
 {% block breadcrumb %}
-  {{ breadcrumb([
-  ]) }}
+  {{ breadcrumb() }}
 {% endblock %}
 
 {% block main_content %}
-
-<header class="page-heading page-heading-with-context">
-    <p class="context">{{ current_user.email_address }}</p>
-
-    <h1>{{ current_user.supplier_name }}</h1>
-</header>
-
-
-<table class="summary-item-body">
+  {% if updated_service_id %}
+    <div class="grid-row">
+    <div class="column-one-whole">
+      <div class="banner-success-without-action">
+        <p class="banner-message">
+          <a href="{{ url_for('.update_service_status', service_id=updated_service_id ) }}">{{ updated_service_name }}</a>
+           is now {{ updated_service_status }}
```

Comment:
The value here needs to be mapped to Public/Private - at the moment it is echoing the DB values enabled/published.